src/template.rs

```diff
@@ -376,23 +376,11 @@ impl<'s> FunctionBuilder<'s, FunctionTemplate> {
     self,
     scope: &mut HandleScope<'s, ()>,
     overload1: &dyn FastFunction,
+    c_fn1: *const CFunctionInfo,
```

Comment:
I think this could be `Option<*const CFunctionInfo>` - if it's not present it will be automatically created for the caller